cmd/xl-v1-list-objects-heal.go

```diff
@@ -325,15 +325,14 @@ func (xl xlObjects) listMultipartUploadsHeal(bucket, prefix, keyMarker,
 		}
 		// Collect uploads until maxUploads limit is reached.
 		for walkResult := range walkerCh {
-			// Ignore errors like errDiskNotFound
-			// and errFileNotFound.
-			if isErrIgnored(walkResult.err,
-				xlTreeWalkIgnoredErrs...) {
-				continue
-			}
-			// For any error during tree walk we should
-			// return right away.
+
+			// For any error during tree walk, except for file not found,
```

Comment:
> if maxUploads > 0 {
>   truncated = false
> }

I see, in that case, I would rename maxUploads to leftUploads, what do you think ?